lib/requester/request-wrapper.js

```diff
@@ -81,62 +76,14 @@ var _ = require('lodash'),
      * @param cb
      */
     setProxy = function(options, cb) {
```

Comment:
separate request options and additional params (from where we mutate request options) sent by us